base/commands/job/job_it_test.go

```diff
@@ -0,0 +1,219 @@
+package job_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	_ "github.com/hazelcast/hazelcast-commandline-client/base/commands"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it"
+)
+
+const jobPath = "testdata/sample-job-1-1.0-SNAPSHOT-all.jar"
+
+func TestJob(t *testing.T) {
+	testCases := []struct {
+		name string
+		f    func(t *testing.T)
+	}{
+		{name: "listNonInteractive", f: listNonInteractiveTest},
```

Comment:
in other tests, we applied suffix with `_`. So maybe, you would like to the same for these test names as well.